feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,334 @@
+    below:
+
+    -   Destination network IPv4-DST-NET with protocol next hop PNH-IPv4.
+    -   Destination network IPv6-DST-NET with protocol next hop PNH-IPv6.
+
+8.  DUT is configured with an IPv4 GUE tunnel with destination IPv4-DST-GUE
+    without any explicit tunnel Type of Service (ToS) or Time to Live (TTL)
+    values.
+
+9.  DUT is configured with the following static routes:
+
+    -   To PNH-IPv4, next hop is the statically configured IPv4 GUE tunnel.
+    -   To PNH-IPv6, next hop is the statically configured IPv4 GUE tunnel.
+
+### PF-1.3.1: IPv4 traffic GUE encapsulation without explicit ToS/TTL configuration on tunnel
+
+ATE action:
+
+*   Generate **IPv4 traffic** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET using a random combination source addresses at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of the packets to *10*.
+
+Verify:
+
+*   Policy forwarding packet counters matches the packet count of traffic
+    generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x80**.
+    *   Inner header ToS is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **9**.
+    *   Inner header TTL is **9**.
+
+### PF-1.3.2: IPv6 traffic GUE encapsulation without explicit ToS/TTL configuration on tunnel
+
+ATE action:
+
+*   Generate **IPv6 traffic** from ATE:Port1 to random IP addresses in
+    IPv6-DST-NET using a random combination source addresses at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of the packets to *10*.
+
+Verify:
+
+*   Policy forwarding packet counters matches the packet count of traffic
+    generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x80**.
+    *   Inner header Traffic Class is **0x80**.
+*   TTL for all GUE encapsulated packets:
+    *   GUE header TTL is **9**.
+    *   Inner header Hop Limit is **9**.
+
+### PF-1.3.3: IPv4 traffic GUE encapsulation with explicit ToS configuration on tunnel
+
+DUT and ATE actions:
+
+*   Re-configure the IPv4 GUE tunnel on the DUT with ToS value *0x60*.
+*   Generate **IPv4 traffic** from ATE:Port1 to random IP addresses in
```

Comment:
Again, I think it's worth clarifying the minimum entropy that you expect here. Something like "ensuring that there are at least X unique source-destination pairs".

Reply:
I would specify the number of 5 tuple flows and a fixed number of packets to send from ATE port 1.  Then the validation should have some number of packets received on each ATE port 2 interface +/- some tolerance.

Reply:
Updated to specify fixed packet count and number of distinct flows.